src/rp2_common/pico_stdio_usb/include/pico/stdio_usb.h

```diff
@@ -102,6 +102,11 @@
 #define PICO_STDIO_USB_CONNECTION_WITHOUT_DTR 0
 #endif
 
+// PICO_CONFIG: PICO_STDIO_USB_DEVICE_SELF_POWERED, Set USB device as self powered device, type=bool, default=0, group=pico_stdio_usb
```

Comment:
I know very little about the USB specs, but is this a "generic USB" thing, or is it definitely specific to just pico_stdio_usb ?

Reply:
I'm not sure that I understand your question. I thought I was clear at the issue.

Reply:
Ahh, sorry if I wasn't very clear. By "generic USB" I think I was trying to ask if something like `PICO_USB_DEVICE_SELF_POWERED` would be a more suitable name than `PICO_STDIO_USB_DEVICE_SELF_POWERED` ?

(But I'm probably just confusing things, so I'll drop out now)